test/e2e/storage/persistent_volumes-local.go

```diff
@@ -53,98 +60,309 @@ const (
 	// TODO: This may not be available/writable on all images.
 	hostBase      = "/tmp"
 	containerBase = "/myvol"
-	testFile      = "test-file"
-	testContents  = "testdata"
-	testSC        = "local-test-storagclass"
+	discoveryDir  = "disks" // 'hostBase + discoveryDir' is the path for volume discovery.
+	// Path to the first volume in the test containers
+	// created via createLocalPod or makeLocalPod
+	// leveraging pv_util.MakePod
+	volumeDir = "/mnt/volume1"
+	// testFile created in setupLocalVolume
+	testFile = "test-file"
+	// testFileContent written into testFile
+	testFileContent = "test-file-content"
+	testSC          = "local-test-storageclass"
+
+	// Following are constants used for provisioner e2e tests.
+	//
+	// testServiceAccount is the service account for bootstrapper
+	testServiceAccount = "local-storage-bootstrapper"
+	// testRoleBinding is the cluster-admin rolebinding for bootstrapper
+	testRoleBinding = "local-storage:bootstrapper"
+	// volumeConfigName is the configmap passed to bootstrapper and provisioner
+	volumeConfigName = "local-volume-config"
+	// bootstrapper and provisioner images used for e2e tests
+	bootstrapperImageName = "quay.io/external_storage/local-volume-provisioner-bootstrap:latest"
+	provisionerImageName  = "quay.io/external_storage/local-volume-provisioner:v1.0.0"
+	// provisioner daemonSetName name, must match the one defined in bootstrapper
+	daemonSetName = "local-volume-provisioner"
+	// provisioner node/pv cluster role binding, must match the one defined in bootstrapper
+	nodeBindingName = "local-storage:provisioner-node-binding"
+	pvBindingName   = "local-storage:provisioner-pv-binding"
+	// A sample request size
+	testRequestSize = "10Mi"
 )
 
-var _ = framework.KubeDescribe("[Volume] PersistentVolumes-local [Feature:LocalPersistentVolumes] [Serial]", func() {
+var _ = framework.KubeDescribe("PersistentVolumes-local [Volume] [Feature:LocalPersistentVolumes] [Serial]", func() {
```

Comment:
Instead of [Volume] we are now using [sig-storage]. There is a merged [pr](https://github.com/kubernetes/kubernetes/pull/48781) for this.